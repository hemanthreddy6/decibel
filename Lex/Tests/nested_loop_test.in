a: int <- 10;

loop a {
    loop 3 {
        loop a - 10 {
            b: string <- "word";
        }
    }
}