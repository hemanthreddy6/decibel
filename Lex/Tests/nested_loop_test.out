IDENTIFIER
:
INT
<-
INT_LITERAL
;
LOOP
IDENTIFIER
{
LOOP
INT_LITERAL
{
LOOP
IDENTIFIER
-
INT_LITERAL
{
IDENTIFIER
:
STRING
<-
STRING_LITERAL
;
}
}
}
